src/pysignalr/client.py

```diff
@@ -3,31 +3,33 @@
 import uuid
 from collections import defaultdict
 from contextlib import asynccontextmanager
-from typing import Any, AsyncIterator, Awaitable, Callable
+from typing import TYPE_CHECKING
+from typing import Any
+from typing import AsyncIterator
+from typing import Awaitable
+from typing import Callable
 
 from pysignalr.exceptions import ServerError
-from pysignalr.messages import (
-    CancelInvocationMessage,
-    CloseMessage,
-    CompletionClientStreamMessage,
-    CompletionMessage,
-    InvocationClientStreamMessage,
-    InvocationMessage,
-    Message,
-    MessageType,
-    PingMessage,
-    StreamInvocationMessage,
-    StreamItemMessage,
-)
-from pysignalr.protocol.abstract import Protocol
+from pysignalr.messages import CancelInvocationMessage
+from pysignalr.messages import CloseMessage
+from pysignalr.messages import CompletionClientStreamMessage
+from pysignalr.messages import CompletionMessage
+from pysignalr.messages import InvocationClientStreamMessage
+from pysignalr.messages import InvocationMessage
+from pysignalr.messages import Message
+from pysignalr.messages import MessageType
+from pysignalr.messages import PingMessage
+from pysignalr.messages import StreamInvocationMessage
+from pysignalr.messages import StreamItemMessage
 from pysignalr.protocol.json import JSONProtocol
-from pysignalr.transport.abstract import Transport
-from pysignalr.transport.websocket import (
```

Comment:
why you do imports from same module on each line rather than use a grouped import?